app/views/microposts/_feed.html.erb

```diff
@@ -0,0 +1,93 @@
+<% @posts.each do |p| %>
+<article class="media">
+  <figure class="media-left">
+    <p class="image is-64x64">
+      <%= link_to gravatar_for(current_user, size: 50)%>
+    </p>
+  </figure>
+  <div class="media-content">
+    <div class="content">
+      <p>
+        <strong><%= p.user.name %></strong>
+        <br>
+        <%= p.content %>
+      </p>
+    </div>
+    <nav class="level is-mobile">
+      <div class="level-left">
+        <a class="level-item">
+          <span class="icon is-small"><%=fa_icon "reply-all"%></span>
+        </a>
+        <a class="level-item">
+          <span class="icon is-small"><%=fa_icon "retweet"%></span>
+        </a>
+        <a class="level-item">
+          <span class="icon is-small">
+          <% pre_like = p.likes.find { |like| like.user_id == current_user.id} %>
+          <% if pre_like %>
+            <%= link_to fa_icon("thumbs-down"), micropost_like_path(p.id, pre_like), method: :delete %>
+          <% else %>
+            <%= link_to fa_icon("thumbs-up"), micropost_likes_path(micropost_id: p.id), method: :post %>
+          <% end %>              
+          </span>
+          <p><%= p.likes.count %> <%= (p.likes.count) == 1 ? 'Like' : 'Likes'%></p>
+        </a>
+      </div>
+    </nav>
+
+    <% @comments.each do |c| %>
+    <% if p.id == c.micropost_id %>
+      <div class="media-right">
+        <button class="micropost delete"></button>
+      </div>
```

Comment:
**[OPTIONAL]**
Finish Implement the post delete functionality. This will improve your page since it will make more similar to real-world applications.

Reply:
Done and tested